### PR TITLE
WooCommerce.com login flow UI update

### PR DESCRIPTION
**Pull request by bgrgicak**
#### Proposed Changes

Update the WooCommerce.com login flow to match with the current WooCommerce branding. 

#### Testing Instructions

- On WooCommerce.test click login
- Replace _https://wordpress.com/_ with _http://calypso.localhost:3000/_ in the url
- Go through the login flow
- Go through the registration flow
- Go through the _Email me a login link flow_
- Confirm that all flows match the screenshots bellow (ignore the outdated authorize screen)

#### Screenshots
Screenshot 2022-07-12 at 10-19-46 Log In — WordPress com
Screenshot 2022-07-12 at 10-20-14 Log In — WordPress com
Screenshot 2022-07-12 at 10-20-45 Log In — WordPress com
Screenshot 2022-07-12 at 10-21-06 Create a site — WordPress com
Screenshot 2022-07-12 at 10-21-42 Log In — WordPress com
Screenshot 2022-07-12 at 10-21-53 Log In — WordPress com
Screenshot 2022-07-12 at 10-22-47 Log In — WordPress com
Screenshot 2022-07-12 at 10-23-00 Log In — WordPress com
Screenshot 2022-07-12 at 10-23-18 Create a site — WordPress com
Screenshot 2022-07-12 at 10-23-31 Log In — WordPress com
Screenshot 2022-07-12 at 10-23-47 Log In — WordPress com
Screenshot 2022-07-12 at 10-24-14 Log In — WordPress com
Screenshot 2022-07-12 at 10-26-17 Authorize WooCommerce com (dev)
Screenshot 2022-07-12 at 10-26-28 Authorize WooCommerce com (dev)

### CODE REVIEW

**Comment by andfinally** [comment]
I'm confused about how to test this. Steps I've tried

- Checked out this branch to my local Calypso and built assets.
- Logged out of woocommerce.test and wordpress.com.
- Clicked "Log in" in woocommerce.test.

But in this case I get the usual wordpress.com UI. Calypso runs on http://calypso.localhost:3000. Code's looking good though!

File: client/layout/masterbar/oauth-client.scss [hunk 1]
```diff
@@ -433,6 +433,62 @@
...
+	a:visited,
+	a:hover,
+	.wp-login__links a:hover,
+	.wp-login__links button:hover,
+	button {
+		font-weight: 500;
+		font-size: $woo-font-size-base;
+		line-height: 28px;
+		text-align: center;
+		color: $woo-purple-60;
```
Comment (andfinally) [inline client/layout/masterbar/oauth-client.scss:485]: Hm, so it looks like they don't worry about a-z order of rules in Calypso!

**Comment by bgrgicak** [comment]
Sorry @andfinally  I forgot to add this part to the testing instructions.
>  Replace _https://wordpress.com/_ with _http://calypso.localhost:3000/_ in the url

**Comment by andfinally** [comment]
Thanks Bero. I'm finding that after I enter my password on

http://calypso.localhost:3000/log-in?client_id=50019&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50019%26state%3D2f398a7ba082768e792dfa13b784f2f9f0de8f1913991bc464c2d7f9d689f031%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.test%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26authorize%3D1%26wccom-from%3D%26calypso_env%3Dproduction



I'm redirected to https://wordpress.com/log-in?client_id=50019&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50019%26state%3D2f398a7ba082768e792dfa13b784f2f9f0de8f1913991bc464c2d7f9d689f031%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.test%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26authorize%3D1%26wccom-from%3D%26calypso_env%3Dproduction and asked to authenticate again – is that expected on local?

**Comment by bgrgicak** [comment]
@andfinally the same thing happens to me. I assume that this is just a local environmental issue.

**Review by andfinally — APPROVED** [review]
Looks great, thanks! 

I'm getting the same login behaviour on trunk, so as you say, it's probably a local issue.

On the acknowledgement page after you request a login link email, the content bleeds off the edge on FF mobile mode. Maybe we need some side padding in `.woo .magic-login`?

image

**Comment by bgrgicak** [comment]
Thank you for helping me with this @andfinally!

I fixed the acknowledgement page.
Screen Shot 2022-07-26 at 12 29 06

**Review by andfinally — APPROVED** [review]
Thanks for the tweaks!

**Pull request merged**